lib/rules/no-unused-vars.js

```diff
@@ -99,7 +99,6 @@ module.exports = {
          * Determines if a given variable is being exported from a module.
          * @param {Variable} variable - EScope variable object.
          * @returns {boolean} True if the variable is exported, false if not.
-         * @private
```

Comment:
There is a lot of rules that do use this keyword and there is a lot of rules that don't, some of the helpers of this particular rule also do and do not.

So it seems we have to choose with or without it, since in my view it is already obvious they are private, i choice the latter.

Trying to make it consistent, what way seems better to you?